vignettes/how-to-bench.Rmd

```diff
@@ -0,0 +1,225 @@
+---
+title: "Benchmarking with decoupleR Tutorial"
+author:
+  - name: "Daniel Dimitrov"
+    affiliation:
+    - Saezlab
+    email: daniel.dimitrov@uni-heidelberg.de
+output: 
+  BiocStyle::html_document:
+    self_contained: true
+    toc: true
+    toc_float: true
+    toc_depth: 3
+    code_folding: show
+package: "`r pkg_ver('decoupleR')`"
+vignette: >
+  %\VignetteIndexEntry{Benchmarking with decoupleR Tutorial}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r chunk_setup, include = FALSE}
+knitr::opts_chunk$set(
+    collapse = TRUE,
+    comment = "#>"
+    )
+```
+
+
+# Prerequsites
+This benchmark setting builds on `decoupleR`, and more specifically the `decouple` wrapper function.
+As such, it requires the decoupleR package to be installed and it is recommended that the user is familiar with the [basics of decoupleR](https://saezlab.github.io/decoupleR/articles/decoupleR.html#basics-1).
+
+The benchmark pipeline requires an input tibble with user-specified settings, benchmark data in the form of a count table and a corresponding metadata table with a priori knowledge of the perturbation experiments or conditions.
+
+The prerequisites used in this tutorial and information about them can be found on [google drive](https://drive.google.com/drive/folders/1p6_9XXuCvArCREakkecSMa5E72aH3wAD) and [zenodo](https://zenodo.org/record/4322914/).
+
+
+```{r load_decoupler, include = TRUE}
+# load dependencies
+library(decoupleR)
+library(ggplot2)
+library(pheatmap)
+library(dplyr)
+library(tibble)
+
+# We load the files directly from Zenodo urls
+# example expression data
+bexample_url = "https://zenodo.org/record/4322914/files/dorothea_bench_example.rds?download=1"
+# full benchmark expression data
```

Comment:
you should elaborate here, that for the vignette itself it is not required to load the entire set or maybe comment this out